Internal test jobs failing

The last 2 CRON jobs seem to have been failed by the same job: [ROS_DISTRO=kinetic _EXTERNAL_REPO='ipa320/cob_calibration_data#c0922cc6f87f31641add6ba4e0fb88c93f9a7c6c' ROS_REPO=ros UPSTREAM_WORKSPACE=file AFTER_SCRIPT='./.travis.xacro_test.sh'](https://github.com/ros-industrial/industrial_ci/blob/33be4ab4e1bc6bc9ca869b2bff11b3447b2ffe1b/.travis.yml#L64)

Output on Travis CI. https://travis-ci.org/ros-industrial/industrial_ci/jobs/294409361#L686
```
Starting  >>> cob_calibration_data                
_______________________________________________________________________________
Errors     << cob_calibration_data:install /root/catkin_ws/logs/cob_calibration_data/build.install.000.log
CMake Error at cmake_install.cmake:169 (file):
  file INSTALL cannot find
  "/root/catkin_ws/src/cob_calibration_data/cob4-16".
make: *** [install] Error 1
```
I skimmed through the upstream but couldn't even find the targeted commit. @ipa-mdl Do you have any idea since you added this https://github.com/ros-industrial/industrial_ci/commit/0d6c5b5b1dfea306b257f3a614ac803b2f4b6b84#diff-354f30a63fb0907d4ad57269548329e3?

--- a/.travis.yml
+++ b/.travis.yml
@@ -61,7 +61,7 @@ env:
     - ROS_DISTRO=kinetic _EXTERNAL_REPO='ros-industrial/industrial_core#kinetic-devel'
     - ROS_DISTRO=kinetic _EXTERNAL_REPO='ros-industrial/motoman_experimental#kinetic-devel' UPSTREAM_WORKSPACE=file BEFORE_SCRIPT='touch $CATKIN_WORKSPACE/src/ros-industrial/industrial_experimental/IRC_v2/CATKIN_IGNORE'
     - ROS_DISTRO=kinetic _EXTERNAL_REPO='ros-controls/ros_control#kinetic-devel' UPSTREAM_WORKSPACE=file ROSINSTALL_FILENAME=ros_control.rosinstall ROS_PARALLEL_TEST_JOBS=-j1
-    - ROS_DISTRO=kinetic _EXTERNAL_REPO='ipa320/cob_calibration_data#c0922cc6f87f31641add6ba4e0fb88c93f9a7c6c' ROS_REPO=ros UPSTREAM_WORKSPACE=file AFTER_SCRIPT='./.travis.xacro_test.sh'
+    - ROS_DISTRO=kinetic _EXTERNAL_REPO='ipa320/cob_calibration_data#indigo_dev' ROS_REPO=ros UPSTREAM_WORKSPACE=file AFTER_SCRIPT='./.travis.xacro_test.sh'
 matrix:
   allow_failures:
     - env: ROS_DISTRO=indigo NOT_TEST_BUILD='true' NOT_TEST_INSTALL='true' POST_PROCESS='I_am_supposed_to_fail'

